S/S2Geometry/build_tarballs.jl

```diff
@@ -3,38 +3,31 @@
 using BinaryBuilder, Pkg
 
 name = "S2Geometry"
-version = v"0.9.0"
+version = v"0.11.1"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/google/s2geometry.git", "a4dddf40647c68cd0104eafc31e9c8fb247a6308"),
-    DirectorySource("./bundled"),
+    GitSource("https://github.com/google/s2geometry.git", "5b5eccd54a08ae03b4467e79ffbb076d0b5f221e"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/s2geometry
-atomic_patch -p1 ../patches/disable-s2testing.patch
 mkdir build && cd build
 cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
     -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
     -DCMAKE_BUILD_TYPE=Release \
     -DBUILD_EXAMPLES=OFF \
+    -DBUILD_TESTS=OFF \
+    -D_USE_MATH_DEFINES \
```

Comment:
This is not how you define macros for the C/C++ compilers, that's only a CMake option.

Reply:
Ah darn, I thought it might pass through.

Reply:
Would that get passed to `make` then?